packages/core/src/config.ts

```diff
@@ -36,9 +36,14 @@ export function normalizeLabel(
  * Go through all the labels in a config and make them
  * follow the same format.
  */
-export function normalizeLabels(config: ConfigObject) {
+export function normalizeLabels(config: ConfigObject): ILabelDefinition[] {
   if (config.labels) {
     const userLabels: ILabelDefinition[] = config.labels.map(normalizeLabel);
+
+    if (config.noDefaultLabels) {
+      return userLabels;
+    }
```

Comment:
I don't understand the `normalizeLabel` function applied to the `config.labels` here. Should we skip it too?

In particular, this part looks suspicious: 

https://github.com/laughedelic/auto/blob/0e921420d2e4a7bc6da32c2255f7107fb7ce97bf/packages/core/src/config.ts#L26-L30

It seems that if a used label has non-`none` release type, it will be replaced with a default label with the same release type 🤔 This would be rather strange behaviour.

Reply:
That gets the colors and descriptions for the default labels. Not exactly sure why is skips the none labels. There might be a test for it.

Reply:
In the context of this change it should be safe to keep. 

What do you think? Would you as a user expect this config option to also disable the descriptions and other metadata?

Reply:
Yep, as a user, I would expect that if I say "don't use default labels", it uses only the information I provide in the `labels` field. I'd be surprised if it added descriptions or colors that I didn't ask for and would be forced to override them.

Reply:
I changed it in 338309b.

I'm open to changing the config option name if you think it helps, e.g. `addDefaultLabels` (true by default) or `defaultLabels` or `normalizeLabels` (IMO less obvious for the users).